website/content/en/docs/helm/reference/watches.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Helm Operator Watches
+linkTitle: Watches
+weight: 20
+---
+
+The Watches file contains a list of mappings from custom resources, identified
+by it's Group, Version, and Kind, to a Helm chart. The Operator
+expects this mapping file in a predefined location: `/opt/helm/watches.yaml`
+
+* **group**:  The group of the Custom Resource that you will be watching.
+* **version**:  The version of the Custom Resource that you will be watching.
+* **kind**:  The kind of the Custom Resource that you will be watching.
+* **chart**: Specifies a chart to be executed. 
+* **watchDependentResources** (default-true): Allows the helm operator to dynamically watch resources that are created by helm.
+* **overrideValues**: Values to be used for overriding Helm chart's defaults. For additional information, please refer [advanced features][advanced-features] document.
```

Comment:
Nit: let's link to the specific section if possible.

That page name is a specific feature of override values, so maybe let's rename that to "Using override values and passing environment variables to the Helm chart"

```suggestion
* **overrideValues**: Values to be used for overriding Helm chart's defaults. For additional information, please refer to [Using override values and passing environment variables to the Helm chart][override-values].
```


And then update the link reference to:

```
[override-values]: /docs/helm/reference/advanced_features/#using-override-values-and-passing-environment-variables-to-the-helm-chart
```